docker/setup.sh

```diff
@@ -89,11 +89,16 @@ function fdb_to_sqlite() {
 update_ini_values
 
 if [[ ! -d "/client" ]]; then
-    echo "Client not found."
-    echo "Did you forget to mount the client into the \"/client\" directory?"
+    echo "[ERROR] Client not found."
+    echo "[ERROR] Did you forget to mount the client into the \"/client\" directory?"
     exit 12
 fi
 
+if [[ "${CLIENT_TYPE@L}" != "packed" && "${CLIENT_TYPE@L}" != "unpacked" && "${CLIENT_TYPE@L}" != "auto" ]]; then
+    echo "[ERROR] Unknown CLIENT_TYPE"
+    exit 15
```

Comment:
Should this be unique? Because you are using the same exit code when `CLIENT_ROOT_DIR` was not detected.

Reply:
Thanks, just quickly skimmed through the code to find out if this code is already used. Will correct it.